<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!--Robot model location -->
  <param name="robot_description" command="$(find xacro)/xacro.py $(find m545_description)/urdf/m545.urdf.xacro" />

  <!-- Self filter node -->
  <node pkg="robot_self_filter" type="self_filter" name="self_filter_R" output="screen">
    <rosparam command="load" file="$(find robot_self_filter)/config/m545.yaml" />
    <param name="cloud_in"                           value="/velodyne_pointsR"/>
    <param name="cloud_out"                          value="/velodyne_pointsR_self_filtered"/>
    <!-- <param name="sensor_frame"                       value="velodyneR_base_link"/> -->
  </node>

  <!-- Self filter node -->
  <node pkg="robot_self_filter" type="self_filter" name="self_filter_L" output="screen">
    <rosparam command="load" file="$(find robot_self_filter)/config/m545.yaml" />
    <param name="cloud_in"                           value="/velodyne_pointsL"/>
    <param name="cloud_out"                          value="/velodyne_pointsL_self_filtered"/>
    <!-- <param name="sensor_frame"                       value="velodyneL_base_link"/> -->
  </node>

</launch>
